src/components/profile-view/profile-view.jsx

```diff
@@ -0,0 +1,214 @@
+// Import components
+import React, { useState } from 'react';
+import MovieCard from '../movie-card/movie-card';
+
+// Import styling
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
+import Container from 'react-bootstrap/Container';
+import Col from 'react-bootstrap/Col';
+
+// Import routing
+import axios from 'axios';
+import { Link } from 'react-router-dom';
+
+const ProfileView = ({ user, movies }) => {
+  const [formData, setFormData] = useState({
+    username: '',
+    email: '',
+    password: '',
+    birthday: '',
+  });
+
```

Comment:
`
componentDidMount() {
		const accessToken = localStorage.getItem('token');
		if (accessToken !== null) {
			this.getUser(accessToken);
		}
	}`

	getUser(token) {
		const username = localStorage.getItem('user');
		axios
			.get(`https://bchanmyflix.herokuapp.com/users/${username}`, {
				headers: { Authorization: `Bearer ${token}` },
			})
			.then((response) => {
				this.setState({
					Username: response.data.Username,
					Password: response.data.Password,
					Email: response.data.Email,
					Birthday: response.data.Birthday,
					FavoriteMovies: response.data.FavoriteMovies,
				});
			})
			.catch(function (error) {
				console.log(error);
			});
	}